src/ngx_http_lua_ssl_certby.c

```diff
@@ -880,6 +880,72 @@ ngx_http_lua_ffi_ssl_server_name(ngx_http_request_t *r, char **name,
 #endif
 }
 
+int
+ngx_http_lua_ffi_ssl_raw_client_addr(ngx_http_request_t *r, char **addr,
+    size_t *addrlen, int *addrtype, char **err)
+{
+#if (NGX_HAVE_UNIX_DOMAIN)
+    struct sockaddr_un *saun;
+#endif
+    ngx_ssl_conn_t *ssl_conn;
+    ngx_connection_t *c;
+    struct sockaddr_in *sin;
+#if (NGX_HAVE_INET6)
+    struct sockaddr_in6 *sin6;
```

Comment:
Style: all the local variable identifiers should be aligned up vertically. Please see other C functions for examples.